bundles/org.openhab.binding.konnected/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -3,23 +3,21 @@
 	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
 	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
-	<!-- This is the Konnected Thing-Type -->
-	<thing-type id="module" extensible="switch,actuator,temperature,humidity">
+	<!-- This is the Konnected Wifi Thing-Type -->
+	<thing-type id="wifi-module" extensible="switch-wifi,actuator-wifi,temperature-wifi,humidity-wifi">
```

Comment:
This is a breaking change - is the renaming of thing-type id and channel type id's needed? In this case it must be mentioned in https://github.com/openhab/openhab-distro/blob/5635e95a953930c2ef01640b392cb8d2a9af4887/distributions/openhab/src/main/resources/bin/update.lst.

Reply:
I don't think it's strictly required. I think it offers clarity on the pro vs the wifi device distinction. Using zone names instead of integers also eases understanding of the configuration.

We probably keep the legacy ids and zones, if needed. I do question how many has active bindings for this, given the number of bugs in the binding.

Reply:
Do you think it's worth keeping the the tech debt to avoid a breaking change?

Reply:
It's always a trade-off between moving forward while providing as much backwards compatibility as possible. If you can provide a note about the need to recreate things after the upgrade, after this PR has been merged, it should be fine. Unfortunately we lack proper migration support.

Reply:
Opened a PR to add an alarm about having to recreate things:
https://github.com/openhab/openhab-distro/pull/1412